examples/16_get_items.rb

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# This example demonstrates how to use get_items to inspect
+# all conversation items including reasoning, web searches, and tool calls.
+
+require_relative "../lib/ai-chat"
```

Comment:
```suggestion
require_relative "../lib/ai-chat"
require "dotenv"
Dotenv.load(File.expand_path("../.env", __dir__))
require "amazing_print"
```

To match the other examples, and allow this to run with the `.env` file.